gcloud/monitoring/test_client.py

```diff
@@ -157,6 +157,120 @@ def test_metric_descriptor_factory(self):
         self.assertEqual(descriptor.description, DESCRIPTION)
         self.assertEqual(descriptor.display_name, '')
 
+    def test_metric_factory(self):
+        TYPE = 'custom.googleapis.com/my_metric'
+        LABELS = {
+            'instance_name': 'my-instance'
+        }
+
+        client = self._makeOne(project=PROJECT, credentials=_Credentials())
+        client.connection = _Connection()   # For safety's sake.
+        metric = client.metric(TYPE, LABELS)
+        self.assertEqual(metric.type, TYPE)
+        self.assertEqual(metric.labels, LABELS)
+
+    def test_resource_factory(self):
+        TYPE = 'https://cloud.google.com/monitoring/api/resources'
+        LABELS = {
+            'instance_id': 'my-instance-id',
+            'zone': 'us-central1-f'
+        }
+
+        client = self._makeOne(project=PROJECT, credentials=_Credentials())
+        client.connection = _Connection()   # For safety's sake.
+        resource = client.resource(TYPE, LABELS)
+        self.assertEqual(resource.type, TYPE)
+        self.assertEqual(resource.labels, LABELS)
+
+    def test_timeseries_factory_gauge(self):
+        import datetime
+        from gcloud._testing import _Monkey
+        import gcloud.monitoring.client
+        METRIC_TYPE = 'custom.googleapis.com/my_metric'
+        METRIC_LABELS = {
+            'status': 'successful'
+        }
+
+        RESOURCE_TYPE = 'gce_instance'
+        RESOURCE_LABELS = {
+            'instance_id': '1234567890123456789',
+            'zone': 'us-central1-f'
+        }
+
+        VALUE = 42
+        TIME1 = datetime.datetime.utcnow()
+
+        client = self._makeOne(project=PROJECT, credentials=_Credentials())
+        client.connection = _Connection()   # For safety's sake.
+        metric = client.metric(METRIC_TYPE, METRIC_LABELS)
+        resource = client.resource(RESOURCE_TYPE, RESOURCE_LABELS)
+
+        # Construct a time series assuming a gauge metric.
+        timeseries = client.time_series(metric, resource, VALUE,
+                                        end_time=TIME1)
+        self.assertEqual(timeseries.metric, metric)
+        self.assertEqual(timeseries.resource, resource)
+        self.assertEqual(len(timeseries.points), 1)
+        self.assertEqual(timeseries.points[0].value, VALUE)
+        self.assertIsNone(timeseries.points[0].start_time)
+        self.assertEqual(timeseries.points[0].end_time, TIME1)
+
+        TIME2 = datetime.datetime.utcnow()
+        # Construct a couple of time series assuming a gauge metric,
+        # using the current time.
```

Comment:
This went wrong. Now the comment in front of the second time series talks about constructing two. Delete "couple of".